Java/other/RecursionCount.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class RecursionCount {
+
+    public static void main(String[] args) {
```

Comment:
Follow the standard Convention
**public static void main(String args[])**

Reply:
@siddharth25pandey sir IntelliJ gives me warning that  `String args[]` is a C-style array declaration. That's why I made it this way. Should I change it?

Reply:
@ritvij14 But for normal jvm, Java Oracle follows the same naming convention.
Anyways, it is not wrong but For maintaining the standardization of code, it is highly recommended.

Reply:
@siddharth25pandey sir oh ok, understood. I have made the changes.